src/App.vue

```diff
@@ -10,9 +10,9 @@ let componentsFactory = new Components({
     language: 'en',
   },
   accessTokenUrl:
-      'https://chargifyapiadaptermock.us-west-2.test.aws.keen.io/merchant/auth?customer=pine-solutions-demo',
+      'https://your.authentication.endpoint.com',
```

Comment:
Maybe add some path to host? e.g. `https://example.com/your-authentication-endpoint` and you can add comment like
```
 'https://example.com/your-authentication-endpoint', // REPLACE WITH YOUR AUTHENTICATION ENDPOINT URL
```

Reply:
done